public/vocables/js/search/searchUtil.js

```diff
@@ -37,7 +37,12 @@ var SearchUtil = (function () {
                                 if (err) return callbackSeries(err);
 
                                 indexes = [];
-                                if (toSources && !Array.isArray(toSources)) toSources = [toSources];
+                                if (toSources) {
+                                    if (!Array.isArray(toSources)) toSources = [toSources];
+                                    toSources.forEach(function (_source) {
+                                        indexes.push(_source.toLowerCase());
+                                    });
+                                }
```

Comment:
should we make indexes a Set ?

would it work to write the following ?

indexes = toSources.map((name) => name.toLowerCase())